convert2rhel/pkghandler.py

```diff
@@ -1144,52 +1111,147 @@ def clean_yum_metadata():
     loggerinst.info("Cached yum metadata cleaned successfully.")
 
 
-def _package_version_cmp(pkg_1, pkg_2):
-    """Compare the version key in a given package name.
+def compare_package_versions(version1, version2):
+    """Compare two EVR versions against each other, as well as
+    the name and architectures
 
-    Consider the following variables that will be passed to this function::
 
-        pkg_1 = 'kernel-core-0:4.18.0-240.10.1.el8_3.x86_64'
-        pkg_2 = 'kernel-core-0:4.18.0-239.0.0.el8_3.x86_64'
+    :param version1: The version to be compared.
+    :type version1: str
+    :param version2: The version to compare against.
+    :type version2: str
```

Comment:
Should we change those parameters to be `package1` and `package2` instead of `version1` and `version2`? I mean, we are passing full package names here, not only versions to be compared. What do you think, @pr-watson?

Reply:
Yes that is more clear to the end users, i'll change this